docs/take-to-prod.md

```diff
@@ -0,0 +1,85 @@
+Taking your .NET for Apache Spark Application to Production
+===
+
+# Table of Contents
+This how-to provides general instructions on how to take your .NET for Apache Spark application to production.
+In this documentation, we will summarize the most commonly asked scenarios when running a .NET for Apache Spark Application.
+You will also learn how to package your application and submit your application with [spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) and [Apachy Livy](https://livy.incubator.apache.org/).
+- [How to deploy your application when you have a single dependency](#how-to-deploy-your-application-when-you-have-a-single-dependency)
+  - [Scenarios](#scenarios)
+  - [Package your application](#package-your-application)
+  - [Launch your application](#launch-your-application)
+- [How to deploy your application when you have multiple dependencies](#how-to-deploy-your-application-when-you-have-multiple-dependencies)
+  - [Scenarios](#scenarios-1)
+  - [Package your application](#package-your-application-1)
+  - [Launch your application](#launch-your-application-1)
+
+## How to deploy your application when you have a single dependency
+### Scenarios
+#### 1. SparkSession code and business logic in the same Program.cs file
+This would be the simple use case when you have `SparkSession` code and business logic (UDFs) in the same Program.cs file and in the same project (e.g. mySparkApp.csproj).
+#### 2. SparkSession code and business logic in the same project, but different .cs files
+This would be the use case when you have `SparkSession` code and business logic (UDFs) in the different .cs files but in the same project (e.g. SparkSession in Program.cs, business logic in BusinessLogic.cs and both are in mySparkApp.csproj).
+
+### Package your application
+Please follow [Get Started](https://github.com/dotnet/spark/#get-started) to build your application.
+
+### Launch your application
+#### 1. Using spark-submit
+Please make sure you have [pre-requisites](https://github.com/dotnet/spark/blob/master/docs/getting-started/windows-instructions.md#pre-requisites) to run the following command.
+```powershell
+%SPARK_HOME%\bin\spark-submit \
+--class org.apache.spark.deploy.dotnet.DotnetRunner \
+--master yarn \
+--deploy-mode cluster \
+--files <some dir>\<dotnet version>\mySparkApp.dll \
+<some dir>\<dotnet version>\microsoft-spark-<spark_majorversion.spark_minorversion.x>-<spark_dotnet_version>.jar \
+dotnet <some dir>\<dotnet version>\mySparkApp.dll <app arg 1> <app arg 2> ... <app arg n>
```

Comment:
`dotnet` is used in this example, but won't this fail in user scenarios where `dotnet` may not be available on their cluster ?

Reply:
Yes I agree, but I have put [pre-requisites ](https://github.com/dotnet/spark/blob/master/docs/getting-started/windows-instructions.md#pre-requisites) in this example.

Or we can also put an example like `mySparkApp args` or `dotnet mySparkApp.dll args` which gives both options depending on cluster environment.